test/calendar-fix-11677.html

```diff
@@ -0,0 +1,63 @@
+
+<!--
```

Comment:
Please rename this file to be `test/calendar-timezone.html` because it's hard to know from the issue id directly.